src/neo4j_graphrag/utils/telemetry.py

```diff
@@ -0,0 +1,8 @@
+import neo4j
```

Comment:
Remeber to add the copyright notice at the top of the file :)

Reply:
19ddec7e4c682600522c21733199c388dfb597b9